src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1351,9 +1402,60 @@ private void getVars(CORINFO_METHOD_STRUCT_* ftn, ref uint cVars, ILVarInfo** va
             // Just tell the JIT to extend everything.
             extendOthers = true;
         }
+
         private void setVars(CORINFO_METHOD_STRUCT_* ftn, uint cVars, NativeVarInfo* vars)
         {
-            // TODO: Debugging
+            if (_localSlotToInfoMap == null && _parameterIndexToNameMap == null)
```

Comment:
I misread the line. Okay for the first question.
With regard to the second question, I'd like to confirm we correctly build/use _parameterIndexToNameMap for both static method and instance method.